<?xml version="1.0"?>
<robot name="irb6400r" version="1.0.0">
    <!-- <group name="robot_only">
        <chain base_link="base_link" tip_link="tool0"/>
    </group> -->
    <!-- <group name="manipulator">
        <chain base_link="base_link" tip_link="carbon_hotwire"/>
    </group> -->
    <group name="manipulator">
        <chain base_link="base_link" tip_link="carbon_hotwire"/>
    </group>

    <!--<group_tcps group="manipulator">
        <tcp name="carbon_hotwire" xyz="0 0 1.861" wxyz="1 0 0 0"/>
    </group_tcps>-->

    <group_state name="home" group="manipulator">
        <joint name="joint_6" value="0"/>
        <joint name="joint_4" value="0"/>
        <joint name="joint_5" value="0"/>
        <joint name="joint_3" value="0"/>
        <joint name="joint_1" value="0"/>
        <joint name="joint_2" value="0"/>
    </group_state>

    <group_opw group="manipulator" a1="0.23999999999999999" a2="-0.22500000000000001" b="0" c1="0.80000000000000004" c2="1.05" c3="1.7250000000000001" c4="2.0059" offsets="0.000000 0.000000 -1.570000 0.000000 0.000000 0.000000" sign_corrections="1 1 1 1 1 1"/>
    
    <!--  c4="0.14499999999999999 -->
    
    <disable_collisions link1="CH_left" link2="CH_right" reason="Never"/>
    <disable_collisions link1="base_link" link2="link_2" reason="Never"/>
    <disable_collisions link1="CH_left" link2="link_6" reason="Never"/>
    <disable_collisions link1="link_2" link2="link_6" reason="Never"/>
    <disable_collisions link1="base_link" link2="link_3" reason="Never"/>
    <disable_collisions link1="carbon_hotwire" link2="link_3" reason="Never"/>
    <disable_collisions link1="CH_right" link2="link_5" reason="Never"/>
    <disable_collisions link1="CH_left" link2="link_5" reason="Never"/>
    <disable_collisions link1="link_2" link2="link_5" reason="Never"/>
    <disable_collisions link1="carbon_hotwire" link2="link_5" reason="Never"/>
    <disable_collisions link1="link_5" link2="link_6" reason="Adjacent"/>
    <disable_collisions link1="link_4" link2="link_5" reason="Adjacent"/>
    <disable_collisions link1="link_3" link2="link_6" reason="Never"/>
    <disable_collisions link1="link_3" link2="link_4" reason="Adjacent"/>
    <disable_collisions link1="carbon_hotwire" link2="link_6" reason="Never"/>
    <disable_collisions link1="link_2" link2="link_3" reason="Adjacent"/>
    <disable_collisions link1="CH_right" link2="link_6" reason="Never"/>
    <disable_collisions link1="link_1" link2="link_2" reason="Adjacent"/>
    <disable_collisions link1="link_3" link2="link_5" reason="Never"/>
    <disable_collisions link1="base_link" link2="link_1" reason="Adjacent"/>
    <disable_collisions link1="CH_right" link2="carbon_hotwire" reason="Adjacent"/>
    <disable_collisions link1="CH_left" link2="carbon_hotwire" reason="Adjacent"/>
</robot>
